src/data/matrix/basic.lean

```diff
@@ -13,6 +13,8 @@ universes u v
 def matrix (m n : Type u) [fintype m] [fintype n] (α : Type v) : Type (max u v) :=
 m → n → α
 
+abbreviation fin_matrix (m n : nat) (α : Type v) := matrix (fin m) (fin n) α
```

Comment:
So far, we've avoided this on purpose. It binds ourselves to one distinguished indexing type, but others might/will show up in practice.

Reply:
Understood, I will drop this commit.